battleship-client/src/main/java/com/epam/solid/nie/client/ui/GameScene.java

```diff
@@ -72,15 +84,10 @@ private void makeEnemyMove(Cell cell1) {
             int x = cell1.getCellX();
             int y = cell1.getCellY();
             Cell cell = playerBoard.getCell(x, y);
-            if (cell.wasShot) {
+            if (cell.wasShot)
                 cell1 = socketServer.receiveEnemyMove();
```

Comment:
I woulnd't use numbers in variables names, and also adding a number solely for the purpose of distinguishing it from another variable is a bad practice, add cell and cell1 different names